ui/src/index.html

```diff
@@ -69,6 +69,11 @@ <h1>Please try reloading.</h1>
       <p>If this keeps happening, contact dreamlab@zenika.com or #dreamlab on Slack.</p>
       <p>You may check the console for more details about the error.</p>
     </div>
+    <div id="errorUnknowUserPage" class="page hidden">
+      <h2>There was an error!</h2>
+      <h1>This user is unknow: <span id="userEmail"></span></h1>
+      <p>If this keeps happening, contact dreamlab@zenika.com or #dreamlab on Slack.</p>
```

Comment:
That's a good start. I think we can do better to explain what is happening. Here's a few suggestions:
- Look at other messages in the app to see how the app talks to users, to take inspiration (for example, the app refers to itself as "I" and to the user as "you")
- Since this is a foreseen "error", I'm not sure we should scare the user with the word "error"
- Make the message explain more precisely what is going on, in terms that the user can understand; users might not be computer savvy
- Maybe we could as far as explaining that this can happen if this is the first day of the employee at Zenika, and that they should checkout the app again tomorrow